new-pipeline/src/dashboards.js

```diff
@@ -714,12 +714,26 @@ function getHumanReadableOptions(filterName, options) {
       options = options.concat([null])
         .concat(badOptions);
     }
-    return options.map(function (option) {
+
+    function cleanArray(actual) {
+	  var newArray = new Array();
+	  for (var i = 0; i < actual.length; i++) {
+	    if (actual[i]) {
+	      newArray.push(actual[i]);
+	    }
+	  }
+	  return newArray;
+	}
+	options = cleanArray(options);
+
+    options = options.map(option => option.replace("aurora", "dev edition"));
+	return options.map(function (option) {
       return option !== null ? [option, option.replace("/", " ")] : null;
```

Comment:
This is addressing things at the right spot, but it's not quite functional yet.
Testing this locally i see that the line separators are missing and clicking the entries in the list doesn't show any data.
I assume this is because now we are returning a list with entries like `["dev edition", "dev edition"]` instead of the expected `["aurora/58", "dev edition"]` or so.

I think we need to leave out the first `...map()`.
Then the second `map()` should return something like `... ? [option, niceName] ...` (where `niceName` is the one that would have "dev edition".
Then take it for a local test and see if it works.